app/views/xml/sections/MainHeader.scala.xml

```diff
@@ -31,6 +31,7 @@
         <fo:inline font-weight="bold">@messages("pdf.personalDetails.nino"): </fo:inline>@model.nino.getOrElse(messages("pdf.personalDetails.noNino"))
         <fo:inline padding-left="2cm" font-weight="bold">@messages("pdf.personalDetails.dob"): </fo:inline>@(model.dob)
     </fo:block>
+    <fo:block margin-bottom="1mm"><fo:inline font-weight="bold">@messages("pdf.personalDetails.title"): </fo:inline>@if(model.name.title.isDefined){@model.name.title.get} else {}</fo:block>
```

Comment:
I think this can just be `model.name.title` as blank options will appear blank anyway